src/main/java/net/minestom/arena/Main.java

```diff
@@ -62,6 +63,9 @@ public static void main(String[] args) {
                 player.setEnableRespawnScreen(false);
             });
 
+            // Prevent dropping
+            handler.addListener(ItemDropEvent.class, event -> event.setCancelled(true));
```

Comment:
This listener should be specific to `LobbyInstance`

Reply:
Fixed in latest commit